lib/backend/k8s/syncer.go

```diff
@@ -172,10 +172,31 @@ func (k *realKubeAPI) getReadyStatus(key model.ReadyFlagKey) (*model.KVPair, err
 
 func newSyncer(kubeAPI kubeAPI, converter converter, callbacks api.SyncerCallbacks, disableNodePoll bool) *kubeSyncer {
 	syn := &kubeSyncer{
-		kubeAPI:         kubeAPI,
-		converter:       converter,
-		callbacks:       callbacks,
-		tracker:         map[string]model.Key{},
+		kubeAPI:   kubeAPI,
+		converter: converter,
+		callbacks: callbacks,
+		trackers: map[string]map[string]model.Key{
+			KEY_NS:  map[string]model.Key{},
```

Comment:
I'm going to pass on this for this PR - it makes some of the little things that work nicely in this version a bit harder / more verbose to do.